Move content from community repo (#4)
open-telemetry/sig-end-user: README.md

# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry End-User SIG

## What is the End-User SIG?

There are two primary goals of the End-User SIG:

> * Foster a sense of vendor-agnostic community for OpenTelemetry end-users to promote learning and adoption. We currently accomplish this through: CNCF Slack channel, panels and blog posts.

Reviewer: ```suggestion
* Foster a sense of vendor-agnostic community for OpenTelemetry end-users to promote learning and adoption. We currently accomplish this through: [CNCF Slack channel](https://cloud-native.slack.com/archives/C01RT3MSWGZ), [panels](https://youtube.com/@otel-official) and blog posts.
```

> * Facilitate the collection and communication of end-user feedback to OpenTelemetry project special interest groups to influence SIG priorities. We currently accomplish this through: creation and maintenance of end-user research assets such as OpenTelemetry Community surveys & Organization and facilitation of structured end-user interview sessions.

Reviewer 2: >OpenTelemetry Community surveys & **o**rganization and facilitation of structured end-user interview sessions

Our group discourages promotion of vendor-specific solutions and desires to keep the focus on OpenTelemetry core technologies, adoption and usage.

## Get Involved

> There are many ways to get involved in our work! We (End-User SIG) meet every two weeks on Thursdays at 10:00 AM PST. Check out the [OpenTelemetry community calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com)

Reviewer: ```suggestion
There are many ways to get involved in our work! We (End-User SIG) meet every two weeks on Thursdays at 10:00 PT/13:00 ET/19:00 CET. Check out the [OpenTelemetry community calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com)
```

> There are many ways to get involved in our work! We (End-User SIG) meet every two weeks on Thursdays at 10:00 PT/13:00 ET/19:00 CET. Check out the [OpenTelemetry community calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com)

Reviewer 2: >OpenTelemetry **C**ommunity calendar

for the Zoom link and any updates to this schedule.

Meeting notes are available as a [public Google doc](https://docs.google.com/document/d/1e-UNZA3Tuno9b53RQbe--whUcO0VIXF3P81oXsrBK6g). If you have trouble accessing the doc, please get in touch on [Slack](https://cloud-native.slack.com/archives/C01RT3MSWGZ).

As an end user, you may want to:

* Reach out to us to tell us your OpenTelemetry adoption story. You can reach out in [#otel-sig-end-user](https://cloud-native.slack.com/archives/C01RT3MSWGZ) to learn more.

> * Take one of our surveys, like our [OpenTelemetry Quarterly Feedback Survey](https://docs.google.com/forms/d/e/1FAIpQLSdKm6oLYRXlZOhEZMVmjoIn4eBToVYNmF6fwpm5GAIipQmPxA/viewform?pli=1)

Reviewer: ```suggestion
* Take one of our surveys, like our [OpenTelemetry/Prometheus Interoperability Survey](https://forms.gle/LLrba9jkzzgfNexN8)
```

I don't think we do the quarterly survey anymore

* Participate in end-user discussions asynchronously on Slack or synchronously via Google Meet.
* Follow this page and [#opentelemetry](https://cloud-native.slack.com/archives/CJFCJHG4Q) to learn about new engagement opportunities as we grow our efforts.
* Encourage other end-users to join and get involved!

> 

Reviewer 2: Delete extra line
